### PR TITLE
PR for DR | Refactor "issues" validations into shared directory

### PR DESCRIPTION
**Pull request by rocio-desantiago**
## Summary 

- Moved `checkValidations` and `uniqueIssue` shared by 996(HLR)/10182(NOD) into `shared/validations/issue.js`
- Moved `selectionRequired` shared by 995(SC)/996(HLR)/10182(NOD) into `shared/validations/issue.js`
- Took out of utils/helpers.js shared by 995(SC)/996(HLR)/10182(NOD): 
     - `someSelected`, `hasSomeSelected`, and `getSelected` because they already exist in
`shared/utils/issues.js`
- Reordered imports so that external libraries are imported first, then VA libraries, and finally from our own forms. Our forms imports are ordered by specific forms directory and shared imports are last.

## Why is this the solution?
- Moving and refactoring these functions into a common shared file will make it easier to make future changes from one distinct file and prevents any one of the forms from being outdated.

## Team
Benefits Team 1

## Related issue(s)
- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#64131

## Testing done
- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

No UI changes

## What areas of the site does it impact?

Decision Review Forms

### Quality Assurance & Testing

- [x] I updated unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### CODE REVIEW

**Review by anniebtran — APPROVED** [review]
overall LGTM — just some small suggestions but nothing blocking!

**Review by anniebtran — CHANGES_REQUESTED** [review]

File: src/applications/appeals/testing/pages/contestableIssues.js [hunk 1]
```diff
@@ -3,16 +3,11 @@ import moment from 'moment';
...
-import { disagreeWith } from '../utils/areaOfDisagreement';
+import { maxIssues } from '../../10182/validations/issues';
 
-import { selectionRequired, maxIssues } from '../../10182/validations/issues';
-
-import { FORMAT_READABLE, FORMAT_YMD, SELECTED } from '../../shared/constants';
-import {
-  hasSomeSelected,
-  getIssueName,
-  getIssueDate,
```
Comment (anniebtran) [inline src/applications/appeals/testing/pages/contestableIssues.js:14]: looks like these should stay too 👀

File: src/applications/appeals/shared/validations/issue.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import { hasDuplicates, hasSomeSelected } from '../utils/issues';
```
[the remark this answers is not in the document]
Comment (rocio-desantiago) [reply src/applications/appeals/shared/validations/issue.js:1]: good catch!

**Review by anniebtran — APPROVED** [review]
👍

**Pull request merged**